### ISSUE TITLE
[1.19.2] Game crashing due to mod after updating

### ISSUE THREAD

**Issue by ShortersOnFire** [lgow/endofherobrine#23]
**Describe the bug**
I recently updated the mod, and since then I can't access the launcher or enter my world (which I have been playing on for some time). I can't delete the mod, or it'll still crash. This has never happened before, so I'm confused on how to fix it.

**To Reproduce**
1. Go to launcher
2. Press play
3. Game crashes
4. Error appears

**Expected behavior**
It's supposed to run normally.

**Screenshots**
When installed-
![Screenshot 2024-01-14 221140](https://github.com/lgow/endofherobrine/assets/156633769/a3c31923-9fb9-4452-83c6-0a794c4da013)

When uninstalled
![Screenshot 2024-01-14 220052](https://github.com/lgow/endofherobrine/assets/156633769/7767e8fd-b41f-4603-bd68-cccfb67eede8)

**Additional context**
I've gone through the files and deleted the mods files that way and through CurseForge, which I've also disabled the mod. No matter what happens it'll still cause the game to crash.

**Comment by Krimatoria** [comment]
Yes, the latest version causes a crash. I have this problem too.
